shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -220,6 +228,18 @@ void onAttach(@NonNull Context context) {
     return activity;
   }
 
+  private FlutterEngineGroup.Options addEntrypointOptions(FlutterEngineGroup.Options options) {
+    String appBundlePathOverride = host.getAppBundlePath();
+    if (appBundlePathOverride == null || appBundlePathOverride.isEmpty()) {
+      appBundlePathOverride = FlutterInjector.instance().flutterLoader().findAppBundlePath();
+    }
+
+    DartExecutor.DartEntrypoint dartEntrypoint =
+        new DartExecutor.DartEntrypoint(
+            appBundlePathOverride, host.getDartEntrypointFunctionName());
+    return options.setDartEntrypoint(dartEntrypoint).setInitialRoute(host.getInitialRoute());
```

Comment:
Here is missing `host.getDartEntrypointArgs()` :-)

Reply:
It makes sense to me that that should be here too.  Good catch, added.